### PR TITLE
Implement highlight a duration when preview line changed

### PR DESCRIPTION
**Pull request by LiuYinCarl**
https://github.com/emacs-vs/goto-line-preview/assets/29608850/e680718d-b030-4582-99d9-202b78d11af2

### CODE REVIEW

**Comment by LiuYinCarl** [comment]
The CI error seems a little confusing and seems to have nothing to do with this code change.

**Comment by LiuYinCarl** [comment]
Implement the suggestion. https://github.com/emacs-vs/goto-line-preview/pull/14#issuecomment-1378977520

File: goto-line-preview.el [hunk 1]
```diff
@@ -48,6 +48,16 @@
   :group 'goto-line-preview
   :type 'hook)
 
+(defcustom goto-line-preview-hl-duration 1
+  "Duration of highlight when change preview line."
+  :group 'goto-line-preview
+  :type 'integer)
```
Comment (jcs090218) [inline goto-line-preview.el:54]: I recommend this accept `nil`, and no highlighting when it's `nil`.

**Review by jcs090218 — COMMENTED** [review]
Thank you for this! And yeah, don't worry about the CI since there are bugs in the windows snapshot! ;)

File: goto-line-preview.el (hunk 1 above)
Comment (LiuYinCarl) [reply goto-line-preview.el:54]: Thanks for your suggestion, you are right. I edit the code, now it can accept nil, is it better now?

Comment (jcs090218) [reply goto-line-preview.el:54]: Thank you!

**Review by jcs090218 — APPROVED** [review]

**Comment by jcs090218** [comment]
LGTM! Thank you!

**Comment by jcs090218** [comment]
Sorry, I've change the color variable `goto-line-preview-hl-color` to a face `goto-line-preview-hl` since it's the standard way to define a custom color. See c98dc626e55195419fc26010e49b3c6d460303bf.

**Pull request merged**